backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/TimeMachineTest.java

```diff
@@ -0,0 +1,227 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
+import static org.mockito.Mockito.*;
+
+import de.bund.digitalservice.ris.norms.application.service.exceptions.ModificationException;
+import de.bund.digitalservice.ris.norms.application.service.exceptions.XmlProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.w3c.dom.Node;
+
+class TimeMachineTest {
+
+  @Test
+  void allRelevantMethodsAreCalled() {
```

Comment:
If it's not too hard, we should list the "relevantMethods". Otherwise the description is equivalent to "works correctly" and nobody's the wiser.